tests/tickets/gh663/manual/1.html

```diff
@@ -0,0 +1,9 @@
+<div id="editor">
+	Apollo 11
+</div>
+
+<script>
+	CKEDITOR.replace( 'editor', {
+		enterMode: CKEDITOR.ENTER_BR
```

Comment:
Is `enterMode: CKEDITOR.ENTER_BR` needed here? It does not affect the issue in any way?